src/sqlfluff/cli/commands.py

```diff
@@ -285,8 +285,17 @@ def dialects(**kwargs):
     is_flag=True,
     help=("Perform the operation regardless of .sqlfluffignore configurations"),
 )
+@click.option(
+    "--parallel",
+    type=int,
+    default=1,
+    help="If set to a value higher than 1, runs SQLFluff in parallel, "
```

Comment:
```suggestion
    help="If set to a value higher than 1, run SQLFluff in parallel, "
```

Just for consistency